admin/meetings/forms.py

```diff
@@ -37,10 +48,9 @@ class MeetingForm(forms.Form):
         required=True,
         widget=forms.TextInput(attrs={'size': '40'}),
     )
-    info_url = forms.CharField(
+    info_url = forms.URLField(
         label='Info url',
-        required=False,
-        widget=forms.TextInput(attrs={'size': '60'}),
+        required=True
```

Comment:
Oh hmm good Q. I thought it was required in the OSF Conferences model, but I was mistaken there. I'll revert!